samples/user-interface/haptics/src/main/java/com/example/platform/ui/haptics/Haptics.kt

```diff
@@ -31,8 +31,8 @@ import com.example.platform.ui.haptics.bounce.BounceRoute
 import com.example.platform.ui.haptics.bounce.BounceViewModel
 import com.example.platform.ui.haptics.expand.ExpandRoute
 import com.example.platform.ui.haptics.expand.ExpandViewModel
-import com.example.platform.ui.haptics.home.HomeRoute
-import com.example.platform.ui.haptics.home.HomeViewModel
+import com.example.platform.ui.haptics.vibrations.VibrationsRoute
```

Comment:
What about "basic"? So `com.example.platform.ui.haptics.basic.BasicRoute`.

Reply:
Yeah Basic could work but let's prefix all with Haptic to make it more clear. Also, since we are modifying it, let's change route to screen. It's a more common standard in compose

Reply:
Basic SGTM

Reply:
Done.
But I couldn't change "route" to "screen" since there's already a composable called `*Screen`.  It seems the naming convention of this sample is `SomethingRoute = SomethingScreen + SomethingViewModel`. I think it's fair.